CONTRIBUTING.md

```diff
@@ -0,0 +1,159 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue, email, or any other method with the owners of this repository
+before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions
+with the project.
+
+## Intellectual Property Rights
+
+All contributions are only accepted under the license in the accompanying
+LICENSE.md file.  This means that when you make a contribution, you are agreeing
+to the terms in the LICENSE.md file, and are certifying that you have the
+necessary rights to make the contribution under the terms of that license.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the
```

Comment:
We might need to make this more specific to the project, as for now it's just a collection of markdown documents.

Reply:
Absolutely true.  I wanted to get this to you ASAP before you had a bunch of other contributors and had to add it in later.  Update it as you see fit.